packages/react-drag-drop/src/components/DragDrop/DragDropSort.tsx

```diff
@@ -0,0 +1,171 @@
+      coordinateGetter: sortableKeyboardCoordinates
+    })
+  );
+
+  const handleDragEnd = (event: DragEndEvent) => {
+    const { active, over } = event;
+    const oldIndex = itemIds.indexOf(active.id as string);
+    const newIndex = itemIds.indexOf(over.id as string);
+    const newItems = arrayMove(items, oldIndex, newIndex);
+    onDrop(event, newItems, oldIndex, newIndex);
+    setActiveId(null);
+  };
+
+  const handleDragStart = (event: DragStartEvent) => {
+    setActiveId(event.active.id as string);
+    onDrag(event, itemIds.indexOf(event.active.id as string));
+  };
+
+  const getDragOverlay = () => {
+    if (!activeId) {
+      return;
+    }
+    const item = getItemById(activeId);
+
+    let content;
+    switch (variant) {
+      case 'DualListSelectorList':
+        content = (
+          <DraggableDualListSelectorListItem key={item.id} id={item.id} {...item.props}>
+            {item.content}
+          </DraggableDualListSelectorListItem>
+        );
+        break;
+      case 'DataList':
+        content = (
+          <DraggableDataListItem key={item.id} id={item.id} {...item.props}>
+            {item.content}
+          </DraggableDataListItem>
+        );
+        break;
+      default:
+        content = (
+          <Draggable useDragButton={variant === 'defaultWithHandle'} key={item.id} id={item.id} {...item.props}>
+            {item.content}
+          </Draggable>
+        );
+    }
+
+    return (
+      <div
+        className={css(styles.draggable, styles.modifiers.dragging)}
+        style={
+          {
+            '--pf-v5-c-draggable--m-dragging--BackgroundColor': 'var(--pf-v5-global--BackgroundColor--100)'
+          } as React.CSSProperties
+        }
+      >
+        {content}
+      </div>
+    );
+  };
+
+  const renderedChildren = (
+    <SortableContext items={itemIds} strategy={verticalListSortingStrategy} id="droppable">
+      {items.map((item: DraggableObject) => {
+        switch (variant) {
+          case 'DualListSelectorList':
+            return (
+              <DraggableDualListSelectorListItem key={item.id} id={item.id} {...item.props}>
+                {item.content}
+              </DraggableDualListSelectorListItem>
+            );
+          case 'DataList':
+            return (
+              <DraggableDataListItem key={item.id} id={item.id} {...item.props}>
+                {item.content}
+              </DraggableDataListItem>
+            );
+          default:
+            return (
+              <Draggable useDragButton={variant === 'defaultWithHandle'} key={item.id} id={item.id} {...item.props}>
+                {item.content}
+              </Draggable>
+            );
+        }
+      })}
+      <DragOverlay>{activeId && getDragOverlay()}</DragOverlay>
+    </SortableContext>
+  );
+
+  return (
+    <DndContext
+      sensors={sensors}
+      collisionDetection={closestCenter}
+      onDragEnd={handleDragEnd}
+      onDragStart={handleDragStart}
+      {...props}
```

Comment:
I think we can prevent autoscrolling by using the [autoscroll props](https://docs.dndkit.com/api-documentation/context-provider#autoscroll). But I'm more concerned with preventing scrolling with keybaord unless they are at the edge of the window, and I want the window to scroll if the user is dragging down past the bottom of the window. It might take more research to figure out how to do that.

Reply:
It looks like autoscroll is disabled already for their keyboard sensor, which internally manages scroll. There might be a different way to customize it for keyboard behavior.

Reply:
let's do that, @kmcfaul i don't think this needs to be blocked by that.

Reply:
Opened https://github.com/patternfly/patternfly-react/issues/9883 to track the scrolling spike